modules/client-configuration/pages/contact-methods-pushssh.adoc

```diff
@@ -112,6 +112,8 @@ When you use the [command]``mgr-ssh-push-init`` tool, the [systemitem]``rhnsd``
 ----
 +
 OPTIONAL: You can remove the [command]``--tunnel`` option, if you do not want to use tunneling.
++
+OPTIONAL: When [command]``ssh_push_sudo_user`` has been defined, add the option [command]``--notty``. This will allow the administrator to enter the ``root`` password. 
```

Comment:
```suggestion
. OPTIONAL: When you have defined [command]``ssh_push_sudo_user``, you can allow use of the root password by adding the [command]``--notty`` option.
```

Reply:
This makes it its own step, rather than tacking it on the previous one, and rewords it to be more active.